File: softuni/Programming Fundamentals/Functions - Exercise/04.odd_and_even_sum.py
number = input()

def solve(num):
    odd_sum = 0
    even_sum = 0

    for i in range(len(num)):
        curr_num = int(num[i])

        if curr_num % 2 != 0:
            odd_sum += curr_num
        else:
            even_sum += curr_num
            
    return f"Odd sum = {odd_sum}, Even sum = {even_sum}"

print(solve(number))

File: JS Front End/Functions and Statements - Exercise/04.odd-and-even-sum.js
function oddAndEvenSum(number) {
    let arr = String(number).split("");
    let oddSum = 0;
    let evenSum = 0;

    arr.forEach((value) => {
        let number = Number(value);
        if (number % 2 === 0) {
            evenSum += number;
        } else {
            oddSum += number;
        }
    });
    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`);
}

oddAndEvenSum(3495892137259234)
